test_awkward_chunks is failing

Reported by @croraf.

```
(venv)
$ make check
./test.sh
python -m tests.test
.................................
----------------------------------------------------------------------
Ran 33 tests in 0.219s

OK
python -m tests.test_js
.....F........
======================================================================
FAIL: test_awkward_chunks (__main__.ESTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/jorendorff/work/rust-frontend/js/jsparagus/tests/test_js.py", line 149, in test_awkward_chunks
    ('script',
AssertionError: Tuples differ: ('scr[164 chars]gn_op', ('div_assign_op', '/=')), ('numeric_literal', '2')))))) != ('scr[164 chars]gn_op', ('div_assign_op',)), ('numeric_literal', '2'))))))

First differing element 1:
('scr[152 chars]ign_op', ('div_assign_op', '/=')), ('numeric_literal', '2')))))
('scr[152 chars]ign_op', ('div_assign_op',)), ('numeric_literal', '2')))))

  ('script',
   ('script_body',
    ('statement_list_single',
     ('expression_statement',
      ('compound_assignment_expr',
       ('identifier_expr', ('identifier_reference', 'x')),
-      ('box_assign_op', ('div_assign_op', '/=')),
?                                         -----

+      ('box_assign_op', ('div_assign_op',)),
       ('numeric_literal', '2'))))))

----------------------------------------------------------------------
```

--- a/tests/test_js.py
+++ b/tests/test_js.py
@@ -152,7 +152,7 @@ class ESTestCase(unittest.TestCase):
                ('expression_statement',
                 ('compound_assignment_expr',
                  ('identifier_expr', ('identifier_reference', 'x')),
-                 ('box_assign_op', ('div_assign_op',)),
+                 ('box_assign_op', ('div_assign_op', '/=')),
                  ('numeric_literal', '2')))))))
 
     def test_can_close(self):

